README.md

```diff
@@ -81,6 +81,10 @@ hooks:
   post_checkout: npm install
   apply: mv .eslintrc .eslintrc.json
   pr_message: echo 'Hey! This PR renames `.eslintrc` to `.eslintrc.json`'
+  issue_message: 'This is the test message'
```

Comment:
As issue_message, issue_labels, state_reason, and state are all associated with issues, I think it makes sense to store these properties in an object set on an issue key:

```yaml
issue:
  message: 'This is the test message'
  labels: ['bug', 'enchancements']
  state_reason: ['completed']
  state: 'closed'
```

Questions:
1. Should we have a title property?  This would allow for a title different from the spec title, which is used for the commit message.
2.  Does message map to issue description?  If so, thoughts on naming it description vs. message?
3.  What is state_reason?  Is this the comment used to close the issue?
4. Thoughts on making state a enum?